docker/nginx.tmpl

```diff
@@ -272,6 +272,20 @@ server {
   }
   {{ end }}
 
+	{{ if eq $host "multipage-frontend.dev.gov.uk" }}
```

Comment:
spacing looks to have slipped here. Do you happen to know if we can do a regex match here or similar so they can be grouped into one?